projects/step-core/src/lib/client/generated/index.ts

```diff
@@ -18,6 +18,7 @@ export type { Attachment } from './models/Attachment';
 export type { AttachmentMeta } from './models/AttachmentMeta';
 export type { AttributeStats } from './models/AttributeStats';
 export type { AutomationPackage } from './models/AutomationPackage';
+export type { AutomationPackageUpdateResult } from './models/AutomationPackageUpdateResult';
```

Comment:
@dvladir This is not related to the BE changes performed. Do you know why we have this change in the generated client? The BE branch is based on master. I might have to rebase it on master?

Reply:
@dvladir thanks for the clarification